src/Maestro/Client/src/MaestroApiCredential.cs

```diff
@@ -51,26 +54,76 @@ public override ValueTask<AccessToken> GetTokenAsync(TokenRequestContext _, Canc
         /// <summary>
         /// Use this for user-based flows (darc invocation from dev machines).
         /// </summary>
-        internal static MaestroApiCredential CreateUserCredential(string barApiBaseUri)
+        internal static MaestroApiCredential CreateUserCredential(string barApiBaseUri, string cachePath)
```

Comment:
Why do we keep passing the `cachePath` across the stack if we could just access `MaestroApiOptions.AUTH_CACHE` directly?

I thought we agreed to remove it from all of the APIs?

Reply:
I was thinking about maybe there being a case where we would need to set it from the infra, but looking back at it its wrong.
My bad, will remove